cub/cub/device/dispatch/dispatch_scan.cuh

```diff
@@ -234,7 +239,8 @@ template <typename InputIteratorT,
                                                                    cub::detail::value_t<InputIteratorT>,
                                                                    typename InitValueT::value_type>,
                                                   cub::detail::value_t<InputIteratorT>>,
-          typename SelectedPolicy = DeviceScanPolicy<AccumT, ScanOpT>>
+          typename SelectedPolicy = DeviceScanPolicy<AccumT, ScanOpT>,
+          bool ForceInclusive     = false>
```

Comment:
Question: we usually put the policy as the last parameter. Why is it necessary to break the design here?

Reply:
Will break backwards compatibility, no?

Reply:
Fair point. However, it's a loud breakage since users would see a compile error (their custom policy cannot initialize a bool) and the fix is really easy. Furthermore, customizing policies strikes me as a feature for a small set of power users. But let's hear @gevtushenko opinion on this.

Reply:
We struggle enough as it is with upsetting our users with unintentionally breaking them. We should not make any intentional breaking changes within a minor version release without extreme justification, and I don't think this case meets that threshold.

Reply:
Alright, so we have an answer :)